README.md

```diff
@@ -1 +1,9 @@
-# product-landing-page-trails
\ No newline at end of file
+# product-landing-page-trails
```

Comment:
The readme is not informative and professional,why don't you add content,listing the technology used in the project and how to runit